File: 2. Java/1. Bronze/4153.java
import java.util.Scanner;

class Main {
    public static void main(String[] args) {
        // Variables
        Scanner sc = new Scanner(System.in);
        int value[] = new int[3];

        while (true) {
            // Take Input & Square Values
            for (int i = 0; i < 3; i++) {
                value[i] = sc.nextInt();
                value[i] *= value[i];
            }
            
            // Loop Break Condition
            if ((value[0] == 0) && (value[1] == 0) && (value[2] == 0)) break;

            // Judgement
            if (value[0] == value[1] + value[2]) System.out.println("right");
            else if (value[1] == value[0] + value[2]) System.out.println("right");
            else if (value[2] == value[0] + value[1]) System.out.println("right");
            else System.out.println("wrong");
        }
    }
}

File: 1. C, C++/1. Bronze/4153.cpp
#include <iostream>

using namespace std;

int main() {
    // Variables
    int value[3];

    while (true) {
        // Take Input & Square Values
        for (int i = 0; i < 3; i++) {
            cin >> value[i];
            value[i] *= value[i];
        }
        
        // Loop Break Condition
        if ((value[0] == 0) && (value[1] == 0) && (value[2] == 0)) break;

        // Judgement
        if (value[0] == value[1] + value[2]) cout << "right" << endl;
        else if (value[1] == value[0] + value[2]) cout << "right" << endl;
        else if (value[2] == value[0] + value[1]) cout << "right" << endl;
        else cout << "wrong" << endl;
    }
}